src/frontend/tests/baseFixtures.ts

```diff
@@ -64,7 +64,8 @@ export const test = baseTest.extend({
         url != 'http://localhost:8000/api/barcode/' &&
         url != 'http://localhost:8000/api/news/?search=&offset=0&limit=25' &&
         url != 'https://docs.inventree.org/en/versions.json' &&
-        !url.startsWith('chrome://')
+        !url.startsWith('chrome://') &&
+        !url.startsWith('http://localhost:5173/node_modules/.vite/')
```

Comment:
That is not a good idea IMO. We will miss a lot of real errors that way. Those are also raised to the user

Reply:
@matmair fair call. It didn't even fix the errors I am seeing...